code/datums/quirks/negative_quirks/claustrophobia.dm

```diff
@@ -7,17 +7,13 @@
 	hardcore_value = 5
 	quirk_flags = QUIRK_HUMAN_ONLY|QUIRK_PROCESSES
 	mail_goodies = list(/obj/item/reagent_containers/syringe/convermol) // to help breathing
+	minimum_process_stat = CONSCIOUS
+	no_process_traits = list(TRAIT_MIND_TEMPORARILY_GONE, TRAIT_FEARLESS, TRAIT_KNOCKEDOUT)
```

Comment:
technically, `minimum_process_stat = CONSCIOUS` should handle this just fine, but I added `TRAIT_KNOCKEDOUT` here just to be 1:1 with the original code.